stl/inc/functional

```diff
@@ -656,26 +656,24 @@ struct _Unforced { // tag to distinguish bind() from bind<R>()
 };
 
 // helper to give INVOKE an explicit return type; avoids undesirable Expression SFINAE
-template <class _Rx, bool = is_void_v<_Rx>>
-struct _Invoker_ret { // selected for _Rx being cv void
-    template <class _Fx, class... _Valtys>
-    static _CONSTEXPR20 void _Call(_Fx&& _Func, _Valtys&&... _Vals) noexcept(_Select_invoke_traits<_Fx,
-        _Valtys...>::_Is_nothrow_invocable::value) { // INVOKE, "implicitly" converted to void
-        _STD invoke(static_cast<_Fx&&>(_Func), static_cast<_Valtys&&>(_Vals)...);
-    }
-};
-
 template <class _Rx>
-struct _Invoker_ret<_Rx, false> { // selected for all _Rx other than cv void and _Unforced
+struct _Invoker_ret { // selected for all _Rx other than _Unforced
     template <class _Fx, class... _Valtys>
     static _CONSTEXPR20 _Rx _Call(_Fx&& _Func, _Valtys&&... _Vals) noexcept(_Select_invoke_traits<_Fx,

```

Comment:
There's a behavioral difference here: we no longer strip cv-qualifiers from the return type when it `is_void`. Should we return `conditional_t<is_void_v<_Rx>, void, _Rx>` to restore that behavior, or is this such an impossible/niche/test suite use case that it's not worth even a minimal hit to throughput?

Reply:
I hope that the new behavior is actually better.

Otherwise `conditional_t` is extra template we fighting against.

probably can do `decltype(auto)` return instead.

Reply:
This is a non-issue since there are no cv-qualified prvalues of non-class type:
```c++
const volatile void f() {}
static_assert(is_same_v<decltype(f()), void>);
```
so nevermind.